src/app/components/audio-recordings/pages/download/download.component.ts

```diff
@@ -42,9 +42,10 @@ interface Model {
   projects?: Project[];
   regions?: Region[];
   sites?: Site[];
-  startedAfter?: Date;
-  finishedBefore?: Date;
-  todEnabled?: boolean;
+  dayFiltering?: boolean;
+  dayStartedAfter?: Date;
+  dayFinishedBefore?: Date;
```

Comment:
i'd use the prefix `date` over `day` here... we are filtering by a date not a day of the week

we've got the input restricted to a date without a time component but what that really translates is just midnight... which is  date with a time